protocols.csv

```diff
@@ -33,5 +33,6 @@ code,	size,	name,	comment
 277,	0,	p2p-stardust,
 275,	0,	p2p-webrtc-star,
 276,	0,	p2p-webrtc-direct,
+280,	0,	webrtc, ICE-lite webrtc transport
```

Comment:
If I understand correctly "ICE-lite" can only be used as a WebRTC server. In other words a browser can not do "ICE-lite", i.e. uses different UDP ports for each connection and does not know its port upfront and thus goes through the whole ICE process. Please correct me if I am wrong.

While I think the "ICE-lite webrtc transport" description is fine for now, I would hope for browsers to be able to re-use the same protocol component (`/webrtc`) in the browser-to-browser use-case, thus we might change **the description** later on.

Reply:
A lite implementation is only appropriate for devices that will *always* be connected to
the public Internet and have a public IP address at which it can receive packets from any
correspondent. ICE will not function when a lite implementation is placed behind a NAT
(RFC8445).

Reply:
Just following up here, the initial hope, namely to reuse `/webrtc` for browser-to-browser (now called webrtc-w3c) is not possible. See https://github.com/libp2p/specs/pull/497 which uses `/webrtc-w3c` instead.